Neos.SiteKickstarter/Classes/Service/SitePackageGeneratorNameService.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Neos\SiteKickstarter\Service;
+
+use Neos\Flow\Reflection\ReflectionService;
+use Neos\SiteKickstarter\Annotation\SitePackageGenerator;
+use Neos\Flow\Annotations as Flow;
+
+class SitePackageGeneratorNameService
+{
+    /**
+     * @var ReflectionService
+     * @Flow\Inject
+     */
+    protected $reflectionService;
+
+    /**
+     * @param string $generatorClass fully qualified namespace
+     */
+    public function getNameOfSitePackageGenerator($generatorClass)
```

Comment:
Honestly I don't like the usage of special annotations here. Wouldn't it be more clear to add `getGeneratorName` / `getGeneratorDescription` to the generatorInterface or use just the className.